### PR TITLE
Merge server code

### PR DESCRIPTION
**Pull request by kskandispersonal**
This is a new branch created using based on standalone-wear, and then reset to upstream:

`
$ git checkout -b new-standalone-wear standalone-wear 
$ git fetch upstream
$ git reset upstream/master 
$ git status 
$ rm log* 
$ git add . 
$ git commit -m "Merge server code"
$ git push origin new-standalone-wear
`

### CODE REVIEW

**Comment by kskandispersonal** [comment]
@jamorham If this looks better, I'll remove the original standalone-wear PR.  Thanks!

File: app/build.gradle [hunk 1]
```diff
@@ -175,7 +175,7 @@ dependencies {
     compile 'com.google.code.gson:gson:2.4'
     compile 'com.squareup.retrofit:converter-gson:2.0.0-beta2'
     compile 'com.squareup.retrofit:retrofit:2.0.0-beta2'
-    compile 'com.getpebble:pebblekit:3.1.0'
```
Comment (jamorham) [inline app/build.gradle]: Example of where this reverts a recent change

**Comment by jamorham** [comment]
This doesn't seem any better that the state of the other PR. See for example the line comment I have added in build.gradle showing where it reverts a recent change.

**Comment by kskandispersonal** [comment]
This is very odd.  Yes, I see that it reverts the change.  Just from the few I looked at, all upstream changes are being reverted to the standalone-wear code wherever there is conflict:(

**Comment by jamorham** [comment]
What happens if you make a new branch and try like `git pull upstream/master` or `git pull upstream/master --rebase` ?

**Comment by kskandispersonal** [comment]
You mean create new branch based on standalone-wear?  Then git pull...?

**Comment by jamorham** [comment]
Well yes, make sure you have a backup in case it goes terribly wrong but you should be able to merge the differences with git pull which will bring your branch up to date based on the earliest parent commit both trees share. Feel free to create as many branches and PRs as you like and I can work off the best one. I really hope we can get something going with this soon because I have a window of time hopefully in the next few days where I would like to try to get this merged.

**Comment by kskandispersonal** [comment]
Nope, git pull... also did not bring in upstream changes correctly:(  It was better (gradle for instance did not revert) but others like BgReading and Calibration (latestValid method) are missing the latest code from upstream!

I will keep trying and hopefully will get it synced up today!

**Pull request closed without merging**